beast/observationmodel/observations.py

```diff
@@ -125,35 +99,33 @@ def setFilters(self, filters):
 
     def getMags(self, num, filters):
         raise Exception('Do not use as magnitudes')
-        return np.array([ self.data[tt][num] - self.distanceModulus for tt in filters])
```

Comment:
We have not been using the distancemodulus in the observation model for sometime.  It was originally used when we took the observed data and put it into absolute units (magnitudes I believe).  Then the models were all computed in absolute units (distance of 10 pc).  And we were comparing the models to the observations in *magnutide* units - which was incorrect!

This is not the correct way to do the comparison given to get the correct noisemodel, we have to put the models at the distance of the galaxy we are observing and compute the uncertainties on the model fluxes using the ASTs.  Thus, we removed the use of the distancemodulus in the observationmodel.  But did not remove passing it to the routines that read in the observed data.

We still use the distance to the galaxy.  It is used to calculate the model fluxes at the distance of the galaxy.   And then we directly compare the observed fluxes to the model fluxes and uncertainties.

So in summary, removing the distancemodulus from the observationmodel code is just remove old code no longer used.  And make things consistent and avoid going back to the old and incorrect way of comparing data to models.  Ok?